test/powershell/Modules/Microsoft.PowerShell.Utility/Start-Sleep.Tests.ps1

```diff
@@ -17,6 +17,13 @@ Describe "Start-Sleep DRT Unit Tests" -Tags "CI" {
         $watch.Stop()
         $watch.ElapsedMilliseconds | Should BeGreaterThan 950
     }
+
+    It "Should work properly when sleeping with ms alias" {
+        $watch = [System.Diagnostics.Stopwatch]::StartNew()
+        Start-Sleep -ms 1000
+        $watch.Stop()
+        $watch.ElapsedMilliseconds | Should BeGreaterThan 950
```

Comment:
In general, we should validate the alias is correct, but shouldn't repeat every test for the parameter and its alias.  `-ms` is a pretty simple case, so it's hard to not repeat the test (using -TestCases is probably better here).  For parameters with more complex behavior, I think it would be sufficient if some cases used the parameter while others used the alias.